packages/admin/cms-admin/src/index.ts

```diff
@@ -27,6 +27,7 @@ export { BuildInformationProvider } from "./common/header/about/build-informatio
 export { useBuildInformation } from "./common/header/about/build-information/useBuildInformation";
 export { Header } from "./common/header/Header";
 export { UserHeaderItem } from "./common/header/UserHeaderItem";
+export { commonImpersonationMessages } from "./common/impersonation/commonImpersonationMessages";
```

Comment:
Is there any use-case where this needs to be imported in a project or another package? 
If not, would not export this. 

```suggestion
```

Reply:
Fixed here: d7bfab67b9463db1d9c30dfc57088503e783efe7
Thank you!